aws-credentials/action.yml

```diff
@@ -0,0 +1,20 @@
+name: 'platform-release-tools-actions'
```

Comment:
Was this action added for a specific reason? It seems like if the keys will be coming from the `inputs`, then the workflow would already be able to call `aws-actions/configure-aws-credentials@v1` directly

Reply:
the idea is to move all the `composite-action` to a global repo and we can look into the option of using it vs not from within the respective workflows of mon-repos or existing ones.

Reply:
Do we have a composite action for aws-credentials? I thought we just call the original action directly